addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -37,6 +41,10 @@
     height: 34px;
 }
 
+.o_ChatterTopbar_buttonSendMessage:not(.o-bordered) {
+    border-left: none;
+}
```

Comment:
I still don't get this one either, but I think it's because the border-left that is overridden here is only present in enterprise? Could this rule be moved there?

Reply:
Yep, right

Reply:
In this case yes then

Reply:
However, I am wondering, actually shouldn't the enterprise code related to this moved here?

Have we tested a bordered in community?

Reply:
Related to what ? here : in `chatter_topbar.scss` of `mail` ?

Reply:
Ok I tested, setting `hasBorder` = false in community works fine, however this variable should be renamed to `hasOuterBorder` or `hasExternalBorder` because the inner border are still there in that case.

Reply:
The prefix is telling you were to put it `o_ChatterTopbar` -> chatter_topbar.scss in mail_enterprise (create folder and file if necessary)

Reply:
But the strange part is that the border that this is overriding should be on the same file theoretically...